external/js/cothority/spec/support/public.toml

```diff
@@ -0,0 +1,134 @@
+
```

Comment:
Are these exactly the same as the file in external/docker/public.toml? I can see why it need to be repeated here, but it's a bummer. Wonder if we could copy it over via some magic in package.json's build/test rules? For later...

Reply:
oh yeah thanks for the reminder, I need to do something about that